### PR TITLE
KOTOR: Add some features for the KotOR Buttons

### PR DESCRIPTION
**Pull request by Nostritius**
when creating a character (either a quick or custom character) in KotOR, the buttons of the steps (like potrait, name ...) behave different then the default button in two ways. They are permanently highlighted, if they are the actual step and have no hovering sound. I added two methods for the KotOR Buttons to enable and disable this features.

### CODE REVIEW

File: src/engines/kotor/gui/widgets/button.h [hunk 1]
```diff
@@ -55,6 +64,6 @@ class WidgetButton : public KotORWidget {
 
 } // End of namespace KotOR
 
-} // End of namespace Engines
+} // End odnf search mate-x*f namespace Engines
```
Comment (DrMcCoy) [inline src/engines/kotor/gui/widgets/button.h]: This is wrong :P

**Comment by DrMcCoy** [comment]
Can you split this into two commits, one for the permanent highlight, one for the sound disabling?

**Comment by DrMcCoy** [comment]
And a third commit for the hover sound, please

**Comment by Nostritius** [comment]
Ok, now i have divided the code and deleted the click sound disabling because i actually don't need it.

**Comment by DrMcCoy** [comment]
Commited it as 048d8fab0...1f448039f, thanks! :)

**Pull request closed without merging**